src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1291,4 +1305,38 @@ export class NetworkAreaDiagramViewer {
             pathElement.setAttribute('d', path);
         }
     }
+
+    private translateEdge(
+        edgeNode: SVGGraphicsElement,
+        offset: Point,
+        side: number
+    ) {
+        const translation = new Point(
+            offset.x - this.initialPosition.x,
+            offset.y - this.initialPosition.y
+        );
+        const transform = DiagramUtils.getTransform(edgeNode);
+        const totalTranslation = new Point(
+            (transform?.matrix.e ?? 0) + translation.x,
+            (transform?.matrix.f ?? 0) + translation.y
+        );
+        edgeNode.setAttribute(
+            'transform',
+            'translate(' +
+                totalTranslation.x.toFixed(2) +
+                ',' +
+                totalTranslation.y.toFixed(2) +
+                ')'
+        );
```

Comment:
this could use the new `moveSvgElement` method?

Reply:
Code changed to using moveSvgElement method